datafusion/sql/src/unparser/expr.rs

```diff
@@ -951,6 +951,87 @@ impl Unparser<'_> {
         }
     }
 
+    /// MySQL requires INTERVAL sql to be in the format: INTERVAL 1 YEAR + INTERVAL 1 MONTH + INTERVAL 1 DAY etc
+    /// https://dev.mysql.com/doc/refman/8.4/en/expressions.html#temporal-intervals
```

Comment:
Let's add the note from the PR description into this comment as well

Reply:
Updated, thank you